lua/wikis/commons/MatchPage/Base.lua

```diff
@@ -212,6 +212,7 @@ function BaseMatchPage:render()
 				mvp = self.matchData.extradata.mvp,
 				opponent1 = self.matchData.opponents[1],
 				opponent2 = self.matchData.opponents[2],
+				gameOneScore = self.games[1].scores,
```

Comment:
isn't scores deprectaed?
should rather read from self.games[1].opponents and extract scores from there (also has status in case of non score results for that game)

Reply:
there is 2 kinds of deprecated:
- deprecated in the way that it is not available in the external api (and marked as deprecated on `Special:LiquipediaDB`)
- deprecated for on wiki usage but still needed for external api (e.g. in match2game all of `.scores`, `.participants`, `.walkover`, `.resulttype` are deprecated and replaced by `.opponents` (plus `.status` for not played matches) instead